frontend/src/app.rs

```diff
@@ -1,11 +1,29 @@
-use crate::landing::LandingPage;
 use yew::prelude::*;
+use yew_router::prelude::*;
+
+use crate::views::home::Home;
+
+#[derive(Clone, PartialEq, Routable)]
+pub enum Route {
+    #[at("/")]
+    Home,
+    #[not_found]
+    #[at("/404")]
+    NotFound,
+}
+
+pub fn switch(routes: Route) -> Html {
+    match routes {
+        Route::Home => html! { <Home /> },
+        Route::NotFound => html! { <h1>{ "404" }</h1> },
```

Comment:
After form submission, add another route here to the poem. It was complaining when I tried to do this and its bedtime